ios/engine/KMEI/KeymanEngineTests/resources/PackageJSON.bundle/package-schema.json

```diff
@@ -0,0 +1,281 @@
+          "items": {
+            "$ref": "#/definitions/startMenuItem"
+          }
+        }
+      },
+      "additionalProperties": false
+    },
+    "startMenuItem": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "filename": {
+          "type": "string"
+        },
+        "arguments": {
+          "type": "string"
+        },
+        "icon": {
+          "type": "string"
+        },
+        "location": {
+          "type": "number"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "name",
+        "filename"
+      ]
+    },
+    "strings": {
+      "type": "object",
+      "patternProperties": {
+        ".": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "files": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/file"
+      }
+    },
+    "file": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "copyLocation": {
+          "type": "number"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "name",
+        "description"
+      ]
+    },
+    "keyboards": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/keyboard"
+      }
+    },
+    "keyboard": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        },
+        "version": {
+          "type": "string"
+        },
+        "oskFont": {
+          "type": "string"
+        },
+        "displayFont": {
+          "type": "string"
+        },
+        "rtl": {
+          "type": "boolean"
+        },
+        "languages": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/keyboardLanguage"
+          }
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "name",
+        "id",
+        "version"
+      ]
+    },
+    "keyboardLanguage": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "id",
+        "name"
+      ]
+    },
+    "info": {
+      "type": "object",
+      "properties": {
+        "website": {
+          "$ref": "#/definitions/infoItem"
+        },
+        "version": {
+          "$ref": "#/definitions/infoItem"
+        },
+        "name": {
+          "$ref": "#/definitions/infoItem"
+        },
+        "copyright": {
+          "$ref": "#/definitions/infoItem"
+        },
+        "author": {
+          "$ref": "#/definitions/infoItem"
+        }
+      },
+      "additionalProperties": false
+    },
```

Comment:
Important note:  the JSON schema used for validation must conform to the draft 4 specification.

Compare against https://api.keyman.com/schemas/package/1.1.0/package.json - I have omitted the following entry beneath `"additionalProperties": false`:

`"required": []`

After said entry is dropped, we're draft 4 compliant.